tests/out/pointer-access.info.ron

```diff
@@ -0,0 +1,355 @@
+(
+    functions: [
```

Comment:
do we need these RON snapshots for pointer-access? They are fairly verbose.
I think we should omit them unless there is a strong reason to include them

Reply:
Okay, sounds good.